loadtest/logstash.conf

```diff
@@ -0,0 +1,11 @@
+input {
+  beats {
+    port => 5044
+  }
+}
+
+output {
+  scalyr {
+    api_write_token => "0/mOvybgzVWVxvH4BKPbbRqBG4KAOx4GTzgEHDYJ3wsI-"
```

Comment:
Should not check in our api token.  Either have a comment to edit this for testing or define the API tokens as CircleCI project vars for automated system test.